### PR TITLE
add README rough draft

### LINKED ISSUE #2: Documentation:  README.md
- [x] Add project name
- [x] Add team members
- [ ] Add schema
- [x] Add required computer setup

### PR DESCRIPTION
**Pull request by ztrokey**
# Description
Adds rough draft of the readme. 

#Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Closes #2

# How Has This Been Tested?
- [x] This has not been tested
- [ ] This has been tested

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,64 @@
...
+Testing
+* [rspec-rails](https://github.com/rspec/rspec-rails)
+* [capybara](https://github.com/teamcapybara/capybara)
+* [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers)
+* [simplecov](https://github.com/simplecov-ruby/simplecov)
+* [webmock](https://github.com/bblimke/webmock)
+* [vcr](https://github.com/vcr/vcr)
+* [Bootstrap](https://github.com/twbs/bootstrap-rubygem)
+
+API Consumption
```
Comment (suzkiee) [inline README.md:52]: Can we add in 'json' and 'jsonapi-serializer`?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,64 @@
...
+## Versions
 
-* Deployment instructions
+- Ruby 2.7.2
 
-* ...
+- Rails 5.2.6
+
+## Important Gems
+Testing
```
Comment (suzkiee) [inline README.md:37]: Can we add `pry`, `factory_bot_rails`, `faker`, and `launchy`?

File: README.md [hunk 3]
```diff
@@ -1,24 +1,64 @@
...
+Run `bundle exec rspec` to run the test suite
+
+## Deployment
+
+See our app in action [here](<!--heroku link here -->)
+
+## Contact
+Created by
+
+* Andrew Shaffer [GitHub](https://github.com/Aphilosopher30) • [LinkedIn](https://www.linkedin.com/in/andrew-shafer-0631ab20a/)
```
Comment (suzkiee) [inline README.md:66]: Omg love this section!

File: README.md (hunk 1 above)
Comment (ztrokey) [reply README.md:52]: Good catch! The deed is done.

File: README.md (hunk 2 above)
Comment (ztrokey) [reply README.md:37]: Good catch! The deed is done.

File: README.md (hunk 3 above)
Comment (ztrokey) [reply README.md:66]: Thanks Suzie, super glad you like it!

File: README.md [hunk 4]
```diff
@@ -1,24 +1,70 @@
...
+* Emmy Morris [GitHub](https://github.com/EmmyMorris) • [LinkedIn](https://www.linkedin.com/in/meaghanmorris1/)
+* Kevin Nguyen [GitHub](https://github.com/denverdevelopments) • [LinkedIn](https://www.linkedin.com/in/kevin-nguyen-59510520a/)
+* Suzie Kim [GitHub](https://github.com/suzkiee) • [LinkedIn](https://www.linkedin.com/in/kim-suzie/)
+* Zach Trokey [GitHub](https://github.com/ztrokey) • [LinkedIn](https://www.linkedin.com/in/zach-trokey/)
```
Comment (denverdevelopments) [inline README.md:70]: Yeah, really great that our GitHub and LinkIn handles are linked, nice Zach

**Review by denverdevelopments — APPROVED** [review]

**Pull request merged**